roles/lib_openshift/src/generate.py

```diff
@@ -27,19 +27,47 @@ def parse_args():
     return parser.parse_args()
 
 
+def fragment_banner(fragment_path, side, data):
+    """Generate a banner to wrap around file fragments
+
+* :param fragment_path: A path to a module fragment
+* :param side: ONE OF: "header", "footer"
+* :param data: A StringIO object to write the banner to
+"""
+    side_msg = {
+        "header": "Begin included fragment: %s",
+        "footer": "End included fragment: %s"
+    }
+    annotation = side_msg[side] % fragment_path
+
+    banner = """
+######################################################################
+# {}
+######################################################################
+""".format(annotation)
+
+    if ("generated" not in fragment_path) and ("license" not in fragment_path):
```

Comment:
@tbielawa explained this was to avoid wrapping generated code and messing with shebangs.